src/mariadb/provision

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+  echo "usage: provision <sandbox-id> <resource-name>"
+  exit 1
+fi
+
+INPUT_DIR=/signadot/provision/input
+OUTPUT_DIR=/signadot/provision/output
+
+# Get values passed to us by Signadot Operator.
+# These tell us which resource name in which Sandbox we should provision.
+SANDBOX_ID=$1
+RESOURCE_NAME=$2
+
+echo "sandbox id: ${SANDBOX_ID}"
+echo "resource name: ${RESOURCE_NAME}"
+
+# Get values passed to us as "params" in the Sandbox's "resources" section.
+DBNAME=$(cat ${INPUT_DIR}/dbname)
+echo "dbname: ${DBNAME}"
+
+# Deploy a temporary DB for this Sandbox in the 'signadot' (helm release) namespace.
+RELEASE_NAME="hotrod-${RESOURCE_NAME}-${SANDBOX_ID}"
```

Comment:
In the hotrod demo, I punted on the problem of resource names with capital letters, which are not allowed in helm release names. I think we should fix that before publishing this official plugin.

One simple mitigation would be to lowercase the resource name here, and say that you can't have two resources in a given sandbox whose names are the same except for case. I don't think we enforce that in the system yet, but it would make sense to start doing so going forward.

```suggestion
RELEASE_NAME="signadot-${RESOURCE_NAME,,}-${SANDBOX_ID}"
```

Reply:
I've created PR #5 as well.  The names don't read well with sandbox id before RESOURCE_NAME.

I agree we should enforce case-insensitive uniqueness of sandbox resource names.